demos/model3d.c

```diff
@@ -109,7 +164,7 @@ Olivec_Canvas vc_render(float dt)
                                 olivec_blend_color(&OLIVEC_PIXEL(oc, x, y), (v<<(3*8)));
                             }
                         }
-                    }
+                    } else if (has_entered) break;
```

Comment:
Exit early if the triangle was previously entered since it isn't possible to re-enter.